dev/docker.ts

```diff
@@ -1958,6 +1958,11 @@ export const completionSpec: Fig.Spec = {
       description: "Manage Docker configs",
       // TODO Subcommands
     },
+    {
+      name: "compose",
+      description: "Contains most of docker-compose features and flags",
+      // TODO Subcommands
+    },
```

Comment:
Based on my previous comment, we can do this:
```suggestion
    {
      name: "compose",
      description: "Contains most of docker-compose features and flags",
      loadSpec: 'docker-compose',
    },
```